CHANGES_NEXT_RELEASE

```diff
@@ -1 +1 @@
-
+- Remove ngsiv1 support for updateAction (#714)
```

Comment:
The following should be removed from https://github.com/telefonicaid/perseo-fe/blob/task/remove_update_ngsiv1/docs/API/plain_rules.md#update-attribute-action

> version: optional, The NGSI version for the update action. Set this attribute to 2 or "2" if you want to use NGSv2 format. 2 by default. It is NOT recommended to use 1.

Reply:
Done in https://github.com/telefonicaid/perseo-fe/pull/717/commits/0c53acfe53164d9651b82eedf06790cf32599645